### PR TITLE
Lk+lr/twitter follow

### PR DESCRIPTION
**Pull request by lindseyreno**
You can follow someone on twitter! We also wrote tests for the new models and the new methods in the feeds controller and application helper.

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def tw_following?(id)
```
Comment (amirahaile) [inline app/helpers/application_helper.rb:2]: Could this be placed in the User model…? Why'd you guys choose this file?

File: app/views/feeds/search_results.html.erb [hunk 2]
```diff
@@ -6,8 +6,13 @@
     <div class="thumbnail">
       <p><%= link_to user.screen_name, "http://twitter.com/#{user.screen_name}" %></p>
       <p><%= image_tag(user.profile_image_url) %></p>
-      <p><%= link_to "Follow", "/", class: "btn btn-success"%></p>
-       <!-- Add a link here once users and sessions are complete -->
+      <% if tw_following?(user.id) %>
```
Comment (amirahaile) [inline app/views/feeds/search_results.html.erb:9]: really like dis :+1:

File: spec/controllers/feeds_controller_spec.rb [hunk 3]
```diff
@@ -37,25 +37,58 @@
...
+  end
 
-      VCR.use_cassette 'controller/instagram_api_search' do
-        get :search_results, instagram_params
-        expect(assigns(:results).first.id).to eq("1105876259")
-      end
+  describe "POST #tw_follow" do
+
+    before(:each) do
+      request.env["HTTP_REFERER"] = "/"
```
Comment (amirahaile) [inline spec/controllers/feeds_controller_spec.rb:65]: woah, this looks scary. what is it?

**Comment by amirahaile** [comment]
I'm ready to merge this. Keeping it open so I'll remember to ask y'all about my comments in the morn'.

File: spec/controllers/feeds_controller_spec.rb (hunk 3 above)
Comment (lindseyreno) [inline spec/controllers/feeds_controller_spec.rb:65]: When you use redirect_to :back you have to give the test a place to go back to.

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (lindseyreno) [inline app/helpers/application_helper.rb:2]: We had it in the Tweet model before we changed the models and then decided it was more of a helper method.

**Pull request merged**